### DISCUSSION TITLE
Miscallenous and Random Issue

### DISCUSSION THREAD

**Discussion by ibnsultan** [thedevdojo/wave#141] [General]
HI, couldn't find an issue section on repo, probably due active development state of the repo, so thought it would not be a bad idea to have a place for comments and stuff on wave 3.

**Comment by bobbyiliev** [comment]
Thanks for starting this discussion! 🙌

**Comment by ibnsultan** [comment]
Suggestion: **Mobile Menu Improvement**
Theme: **Drift**

I think what's on the select menu should be what's first visible since it's the primary menu, and the utility menu should go to the select area

![image](https://github.com/user-attachments/assets/d1eb1ebc-0efa-4309-b2c9-4833d7ca3e4a)


cc @tnylea @bobbyiliev

**Comment by webtechlabs** [comment]
Is there anyone else having db:seed issues?  It isn't populating the database at all.  Still zero records.  I'm not seeing any error messages in the terminal.  But when doing them manually one by one (PITA) I did notice that there are some left over migrations from Wave 2 and at least one row entry didn't match up exactly with the Wave 3 table schema.
